include/qbsp/map.hh

```diff
@@ -194,8 +201,17 @@ void LoadMapFile(void);
 mapentity_t LoadExternalMap(const char *filename);
 void ConvertMapFile(void);
 
+struct extended_tx_info_t {
+    bool quark_tx1 = false;
+    bool quark_tx2 = false;
+    
+    int contents = 0;
+    int flags = 0;
+    int value = 0;
+};
+
```

Comment:
This was moved to the header in preparation for it needing to be used for FindTexinfo and friends, for Q2, since the flags/value are important to matching texinfo structs